test/base/bitwise.js

```diff
@@ -75,30 +75,30 @@ describe('bitwise functions async training', () => {
     let not = [{input: [0], output: [1]},
                {input: [1], output: [0]}];
     testBitwiseAsync(not, 'not', done);
-  }).timeout(10000);
+  }).timeout(15000);
```

Comment:
why the change?

Reply:
I had a test that timed out the first time.  I am actually thinking we should get rid of the async tests as much as possible (now that the training ticks are shared)  They really delay the test suit

Reply:
definitely should change up how we test the async. however, that should be done in a separate PR and I'd say just revert this back to limit unnecessary changes of this PR.

Reply:
👍